pom.xml

```diff
@@ -1,28 +1,22 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>org.zalando.stups.build</groupId>
-        <artifactId>stups-parent</artifactId>
-        <version>6</version>
-        <relativePath />
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
```

Comment:
please keep in mind, that changing this parent pom will remove the Maven Central deploy configuration. It needs to be added manually here in the pom again.

Reply:
Also, this is a breaking change, since we will no longer support Spring Boot 1.x. The new version should bump the major version.

Reply:
Applied in #26